charts/openbao/values.yaml

```diff
@@ -1409,3 +1409,40 @@ extraObjects: []
 #     kind: ConfigMap
 #     metadata:
 #       name: cert-manager-configmap-{{ include "some-other-template" }}
+
+# Snapshot Agent Configuration
+snapshotAgent:
+  # whether or not to enable the snapshot agent cronjob
+  enabled: false
+  # extra Annotations for the job
+  annotations: {}
+  # schedule of the cronjob
+  schedule: "*/5 * * * *"
```

Comment:
The default of every 5 minutes might a bit too frequent, wdyt? I think every 15mins / 20 mins would be fine for a default.